code/modules/projectiles/projectile/special.dm

```diff
@@ -94,8 +94,7 @@
 			M.bodytemperature = temperature
 			if(temperature > 500)//emagged
 				M.adjust_fire_stacks(0.5)
-				M.on_fire = 1
-				M.update_icon = 1

```

Comment:
@Crazylemon64 i hope you are kidding because the update_icon var shouldn't exist let alone ever be used